src/app/workflow/launch-third-party/dialog/launch-to-codespace-dialog.component.html

```diff
@@ -0,0 +1,35 @@
+<h1 mat-dialog-title>Launch this {{ term }} to a GitHub Codespace?</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+  <p>
+    You are about to launch the {{ term }} <b>{{ path }}</b> in a GitHub Codespace. To configure the Codespace, GitHub reads a
+    <a href="https://containers.dev/">development container</a> ("dev container") file from the repository, which specifies the files to use
```

Comment:
Tried link and it redirects to https://containers.dev, without a trailing slash, might was well as remove it from here as well.